src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1270,8 +1270,8 @@ public static void ContainsNoMatch_StringComparison()
                         firstSpan.ToString().StartsWith(secondSpan.ToString(), StringComparison.CurrentCulture),
                         firstSpan.Contains(secondSpan, StringComparison.CurrentCulture));
                     Assert.Equal(
-                        firstSpan.ToString().StartsWith(secondSpan.ToString(), StringComparison.CurrentCulture),
-                        firstSpan.Contains(secondSpan, StringComparison.CurrentCulture));
+                        firstSpan.ToString().StartsWith(secondSpan.ToString(), StringComparison.CurrentCultureIgnoreCase),
```

Comment:
this was a copy paste error (`StringComparison.CurrentCulture` was tested twice)